knip.jsonc

```diff
@@ -0,0 +1,33 @@
+{
+    "$schema": "https://unpkg.com/knip@3/schema.json",
+    "entry": [
+        "Herebyfile.mjs",
+        // Knip does not understand Herebyfile.mjs (a new Knip plugin could probably handle it), so we add the entry files manually:
```

Comment:
Just as a general note, I doubt a plugin would be a good idea for this sort of thing. The Herebyfile is just random code and could produce files however it likes; there's no specific schema to it besides "exports tasks".

Reply:
Cool, I didn't even look at Hereby at all yet.

Reply:
I would just drop this comment at this point; like I said above, a plugin is unlikely to actually help here I think.